examples/0xx_parallel_computation.py

```diff
@@ -0,0 +1,117 @@
+"""Example: Compare parallel interfaces
+"""
+
+from time import perf_counter as timerpc
+
+import numpy as np
+
+from floris import (
+    FlorisModel,
+    TimeSeries,
+    WindRose,
+)
+from floris.parallel_floris_model import ParallelFlorisModel as ParallelFlorisModel_orig
+from floris.parallel_floris_model_2 import ParallelFlorisModel as ParallelFlorisModel_new
+
+
+if __name__ == "__main__":
```

Comment:
Something I've been wondering is there a line we could include within ParallelFlorisModel_new to detect whether or net it has been initialized following a if __name__ == "__main__":, and if not giving a helpful warning